pkg/api/types.go

```diff
@@ -331,12 +348,24 @@ const (
 	// used for PersistentVolumes that are not available
 	VolumePending PersistentVolumePhase = "Pending"
 	// used for PersistentVolumes that are not yet bound
+	// Available volumes are held by the binder and matched to PersistentVolumeClaims
 	VolumeAvailable PersistentVolumePhase = "Available"
 	// used for PersistentVolumes that are bound
 	VolumeBound PersistentVolumePhase = "Bound"
 	// used for PersistentVolumes where the bound PersistentVolumeClaim was deleted
 	// released volumes must be recycled before becoming available again
+	// this phase is used by the persistent volume claim binder to signal to another process to reclaim the resource
 	VolumeReleased PersistentVolumePhase = "Released"
+	// used for PersistentVolumes that have been recycled by a recycler and can be made available again to new PersistentVolumeClaims
+	// recycled volumes have been scrubbed of their data but the recycler does not understand claim matching.
+	// this phase is used by a recycler to communicate to the binder that the volume is ready to be consumed again as a fresh resource
+	// it is the binder's responsibility to make the volume Available again so that it can be claimed by a new user
+	VolumeRecycled PersistentVolumePhase = "Recycled"
```

Comment:
I know you're trying here, but I just don't see the point of this phase.  The only thing that can happen to a recycled volume would be to make it Active, right?  (I mean you could delete it but what is the point of recycling?)  Is there some race or synchronization trick  that I am not comprehending that warrants putting this in as a distinct state?

At this point I am asking for a discussion, not more comments :)